### PR TITLE
Add TypeScript typings

### PR DESCRIPTION
**Pull request by alexander-fenster**
Hi @hgcummings,

Thanks for the library! I'm using it in my small hobby project that is written in TypeScript, and I needed to write these typings to make it possible. I realize it's a 4 years old project. If you're willing to publish an update with these types to NPM, I will really appreciate it! Otherwise, I can add this `.d.ts` to DefinitelyTyped repository if you prefer it that way.

Thank you!

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+export interface RenderOptions {
+  foreground: Color;
+  background: Color;
+  scale?: number;
+}
+
+export const fonts: {
+  sevenPlus: Font;
+  slimbers: Font;
```
Comment (hgcummings) [inline index.d.ts]: Typo here: the font name is "slumbers"

**Review by hgcummings — CHANGES_REQUESTED** [review]
Hi Alexander. Thanks for the PR!

Looks good and I'm happy to merge it. Just one typo to fix first.

File: index.d.ts (hunk 1 above)
Comment (alexander-fenster) [reply index.d.ts]: Oops, fixed, thank you!

**Review by hgcummings — APPROVED** [review]

**Comment by hgcummings** [comment]
Merged. Thank you for the PR!

**Comment by alexander-fenster** [comment]
@hgcummings Thank you Harry! Will you have a chance to release this to npm? Thank you!

**Comment by hgcummings** [comment]
Yep, a bit busy yesterday but will publish a new version at some point today (UK time, so might be overnight for you).

**Comment by hgcummings** [comment]
Published v1.6.0

Let me know if you encounter any issues using the new release.

**Pull request merged**